### ISSUE TITLE
Trouble importing dask_deltatable

### ISSUE THREAD

**Issue by scharlottej13** [dask-contrib/dask-deltatable#81]
Hi! I'm having some trouble importing dask-deltatable. When I import I get the following error for versions 0.3.1 and 0.3.0 (didn't try other versions after that).

```python
import dask_deltatable as ddt

ImportError: cannot import name 'MAX_SUPPORTED_WRITER_VERSION' from 'deltalake.writer' (/Users/sarahj/mambaforge/envs/test/lib/python3.11/site-packages/deltalake/writer.py)
```

It looks like this is fixed for the latest commit (if I instead do a pip install for the latest commit, I do not get this error).

**Comment by jacobtomlinson** [comment]
Yeah it looks like this was fixed in #76 but hasn't been included in a release yet. Given that CI is passing on main I'm going to get a release out now.

**Comment by jacobtomlinson** [comment]
I've released the current state of main as `0.3.2` but I'm not sure this fixes the issue as the builds are failing. 

We now see `ImportError: cannot import name 'PYARROW_MAJOR_VERSION' from 'deltalake.writer'`. It looks like more recent versions of `deltalake` have renamed some constants. I'll get a PR open to fix.

**Comment by jacobtomlinson** [comment]
This should be resolved now with the `v0.3.3` release.

**Comment by scharlottej13** [comment]
So speedy! Yup that works for me. Thanks @jacobtomlinson!
